modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,260 @@
+* Log in as a user with `cluster-admin` privileges.
+* Run `setsebools container_use_devices=true` as root on all nodes.
+
+.Procedure
+
+. Create the following `SriovNetworkNodePolicy` object, and then save the YAML in the `intel-dpdk-node-policy.yaml` file.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: intel-dpdk-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: intelnics
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  priority: <priority>
+  numVfs: <num>
+  nicSelector:
+    vendor: "8086"
+    deviceID: "158b"
+    pfNames: ["<pf_name>", ...]
+    rootDevices: ["<pci_bus_id>", "..."]
+  deviceType: vfio-pci <1>
+----
+<1> Specify the driver type for the virtual functions to `vfio-pci`.
++
+[NOTE]
+=====
+See the `Configuring SR-IOV network devices` section for a detailed explanation on each option in `SriovNetworkNodePolicy`.
+
+When applying the configuration specified in a `SriovNetworkNodePolicy` object, the SR-IOV Operator may drain the nodes, and in some cases, reboot nodes.
+It may take several minutes for a configuration change to apply.
+Ensure that there are enough available nodes in your cluster to handle the evicted workload beforehand.
+
+After the configuration update is applied, all the pods in `openshift-sriov-network-operator` namespace will change to a `Running` status.
+=====
+
+. Create the `SriovNetworkNodePolicy` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f intel-dpdk-node-policy.yaml
+----
+
+. Create the following `SriovNetwork` object, and then save the YAML in the `intel-dpdk-network.yaml` file.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: intel-dpdk-network
+  namespace: openshift-sriov-network-operator
+spec:
+  networkNamespace: <target_namespace>
+  ipam: |-
+# ... <1>
+  vlan: <vlan>
+  resourceName: intelnics
+----
+<1> Specify a configuration object for the IPAM CNI plugin as a YAML block scalar. The plugin manages IP address assignment for the attachment definition.
++
+[NOTE]
+=====
+See the "Configuring SR-IOV additional network" section for a detailed explanation on each option in `SriovNetwork`.
+=====
++
+An optional library, app-netutil, provides several API methods for gathering network information about a container's parent pod.
+
+. Create the `SriovNetwork` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f intel-dpdk-network.yaml
+----
+
+. Create a YAML file `tap-example.yaml` defining a network attachment definition, such as in the following example:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+ name: tap-one
+ namespace: <target_namespace> <1>
+spec:
+ config: '{
+   "cniVersion": "0.4.0",
+   "name": "tap",
+   "plugins": [
+     {
+        "type": "tap",
```

Comment:
I think we need to add more infor about tap plugin. May be you can add link to the tap documentation ?

Reply:
Once I get this submitted and the other related ones I was going to revisit and create a separate PR to link various sections that have related information